packages/components/src/components/bal-list/bal-list.scss

```diff
@@ -394,7 +443,9 @@ bal-card > bal-list.p-0 {
   }
 
   &.is-open bal-icon.trigger-icon {
-    transform: rotate(45deg);
+    bal-list:not(.is-in-main-nav) & {
+      transform: rotate(45deg);
```

Comment:
what is rotating ?

Reply:
I'm not sure, that was from you. Madalina just added the previous check for the main nav